docs/index.rst

```diff
@@ -1,26 +1,32 @@
-Metagenome Assembly Workflow (v1.0.2)
-========================================
+Metagenome Assembly Workflow (v1.0.7)
+=====================================
 
 .. image:: workflow_assembly.png
    :scale: 60%
    :alt: Metagenome assembly workflow dependencies
-   
+
 Workflow Overview
 -----------------
 
-This workflow takes in paired-end Illumina reads in interleaved format and performs error correction, then reformats the interleaved file into two FASTQ files for downstream tasks using bbcms (BBTools). The corrected reads are assembled using metaSPAdes. After assembly, the reads are mapped back to contigs by bbmap (BBTools) for coverage information. The .wdl (Workflow Description Language) file includes five tasks, *bbcms*, *assy*, *create_agp*, *read_mapping_pairs*, and *make_output*.
+This workflow takes in paired-end Illumina short reads or paired-end PacBio long reads in interleaved format and performs error correction, then reformats the interleaved file into two FASTQ files for downstream tasks using bbcms (BBTools). The corrected reads are assembled using metaSPAdes. After assembly, the reads are mapped back to contigs by bbmap (BBTools) for coverage information. The `.wdl` (Workflow Description Language) file includes five tasks: *bbcms*, *assy*, *create_agp*, *read_mapping_pairs*, and *make_output*.
```

Comment:
pacbio is not paired-end

Reply:
this paragraph needs to be re-written. The long reads workflow doesn't use metaSPAdes or bbcms.